Bottom: 3dbd82583b08701b2cb77410d637c7620ab3f11a
Top:    e7d4d03d082c95eafddd2b042a4fc804e0671f67
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-02 18:43:08 +0000

Make --enable-gl default when building the GTK HID


---

diff --git a/configure.ac b/configure.ac
index 50370f5..0f11e4a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -360,8 +360,12 @@ fi
 AC_MSG_CHECKING([for whether to use GL drawing])
 AC_ARG_ENABLE([gl],
 [  --enable-gl           Enable GL drawing (with GTK HID)],
-[],[enable_gl=no])
-
+[],[
+	case " $with_gui " in
+		*\ gtk\ *) enable_gl=yes;;
+		* ) enable_gl=no;;
+	esac
+])
 AC_MSG_RESULT([$enable_gl])
 AM_CONDITIONAL(USE_GL, test x$enable_gl = xyes)
